templates/example_MPH_plots.ipynb

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
<p>If these are the defaults, we should define them and pass them in, rather than commenting them out and having two different versions of the function.</p><p> </p><p>Also, given that this will now be getting saved in the bin_file_dir, we should name this something besides pulse_height (qc_pulse, moly_pulse, ect) since the normalization function looks for all csv files with pulse_height in the name</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>